dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -93,19 +93,71 @@ For the data types above:
 ##############
 
 - Change the axes that are displayed by selecting the relevant ``X`` and ``Y`` axes in the top left. This is more interesting for higher dimension data.
+
+4. Non-orthogonal view
+######################
+
 - For the ``SXD_MD_nonortho`` workspace, the non-orthogonal view button (see below) should be enabled - clicking this should also turn on grid lines. When viewing the H and K axes you should see the gridlines are no longer perpendicular to each other.
 
 .. figure:: ../../../../docs/source/images/wb-sliceviewer51-nonorthobutton.png
    :class: screenshot
    :align: center
 
-4. Cursor Tracking
+- Check that sliceviewer gets the projection correct for an ``MDHisto`` object from a non-axis aligned cut.
```

Comment:
Can you reword this one line to be less sciencey? I think it currently means nothing to most of our dev testers as what does the "correct projection" look like? The other lines are well explained!

Reply:
Ah I didn't intend that as an instruction itself, the "test" for getting the correct projection matrix/basis vectors as columns from the workspace is outlined in more detail in the lines below. I will clarify this and replace projection with basis vectors which is a term more people will be familiar with.